pkg/resmgr/flags.go

```diff
@@ -66,6 +67,7 @@ func init() {
 		"Configuration used to override the one stored in the cache. Disables the agent.")
 	flag.StringVar(&opt.ForceConfigSignal, "force-config-signal", "SIGHUP",
 		"Signal used to reload forced configuration.")
+	flag.BoolVar(&opt.CacheOutputEnable, "cache-output-enable", false, "Support cache output via HTTP handler.")
```

Comment:
nitpick: Could we find/use a more descriptive command line option name for this ? Maybe `--enable-cache-http-query`, but I'm not good with names, so this is just a vague attempt at a name.